lib/job-handlers.js

```diff
@@ -2,7 +2,7 @@ var _ = require('underscore');
 var JobManager = require('./job-manager');
 var ObserverManager = require('./observer-manager');
 var WebsocketEndpoint = require('./websocket-endpoint');
-var logger = require('log4js').getLogger('jobs-handler'); //jshint unused:false
+var logger = require('log4js').getLogger('jobs-handler'); // eslint-disable-line
```

Comment:
Unrelated to this PR but not sure why logger needs to be required if its not used. @mstemm?